ui/panels-plugin/src/plugins/scatterplot/Scatterplot.tsx

```diff
@@ -60,7 +60,9 @@ export function Scatterplot(props: ScatterplotProps) {
       axisPointer: {
         type: 'cross',
       },
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
       formatter: function (params: any) {
```

Comment:
This [threw lint errors](https://github.com/perses/perses/actions/runs/7824675331/job/21347614114) on my PR, but should have been caught in #1672, wonder if something is up with PR checks 😕 (added TODO for changing `any` to import directly from ECharts types)

Reply:
Oh I guess no-explicit-any is just a warning so won't fail that check, but probably should be an error